manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -8,8 +8,10 @@ meta:
   api_templates:
   - name: consul_agent
     release: (( grab meta.consul_templates.consul_agent.release ))
+    consumes: {consul: nil}
```

Comment:
Not sure, but they provide a scoped way to share connection details between "jobs" in a deployed. For instance, you can deploy multiple different consul clusters, and then consume the details from one to the other. 

It also prevents having to repeat the properties of one server in the clients.